Sources/Datadog/FeaturesIntegration/CrashReporting/CrashReportingWithRUMIntegration.swift

```diff
@@ -73,14 +88,63 @@ internal struct CrashReportingWithRUMIntegration: CrashReportingIntegration {
             rumEventOutput.write(rumEvent: rumError)
             rumEventOutput.write(rumEvent: rumView)
         } else {
+            // We know it is too late for sending RUM view to previous RUM session as it is now stale on backend.
+            // To avoid inconsistency, we only send the RUM error.
             let rumError = createRUMError(from: crashReport, and: lastRUMViewEvent, crashDate: realCrashDate)
             rumEventOutput.write(rumEvent: rumError)
         }
     }
 
+    /// If the crash occurred before starting RUM session (after initializing SDK, but before starting the first view) we don't have any session UUID to associate the error with.
+    /// In that case, we consider sending this crash within a new, single-view session: either "ApplicationLaunch" view or "Background" view.
+    private func sendCrashReportToNewSession(
+        _ crashReport: DDCrashReport,
+        crashContext: CrashContext,
+        using currentTimeCorrection: DateCorrection
+    ) {
+        // We can ignore `sessionState` for building the rule as we can assume there was no session sent - otherwise,
+        // the `lastRUMViewEvent` would have been set in `CrashContext` and we could be sending the crash to previous session
+        // through `sendCrashReportToPreviousSession()`.
```

Comment:
This turns out to be not true, as discussed in [this comment in previous PR](https://github.com/DataDog/dd-sdk-ios/pull/690/files#r772978874). I will update it to also take `crashContext.lastRUMSessionState` into account as now we're not handling all off-view background crashes properly. More precisely: when crash happens as the very first event after sending app to background, it won't be send in "Background" view.

Reply:
Done 👍, I also updated the diagram in PR description accordingly.